modules/client_gui.py

```diff
@@ -0,0 +1,198 @@
+    canvas = Canvas(window_handle, width=1000, height=600)
+    display_image(canvas, table_img)
+    return canvas, table_img
+
+
+def display_card(canvas, table_img, index, player_index, card_im):
+    """Add card to table canvas."""
+    x_loc = 350 + index * (CARD_WIDTH + 28)
+    y_loc = 100 + player_index * 280
+    table_img.paste(card_im, (x_loc, y_loc))
+    display_image(canvas, table_img)
+
+
+def display_name(canvas, table_img, name, player_index):
+    """Add names to the table."""
+    x_loc = 450
+    y_loc = 50 + player_index * 450
+    font = ImageFont.truetype('arial.ttf', 40)
+    draw = ImageDraw.Draw(table_img)
+    draw.text((x_loc, y_loc), name, font=font, fill=(255,128,128,255))
+    display_image(canvas, table_img)
+
+
+def display_chip_tray(canvas, table_img, value, player_index):
+    """Display the chip tray."""
+    x_loc = 50
+    y_loc = 50 + player_index * 400
+    tray_img = make_chip_tray_image(value)
+    table_img.paste(tray_img, (x_loc, y_loc))
+    display_image(canvas, table_img)
+
+def display_pot_tray(canvas, table_img, value):
+    """Display the pot tray."""
+    x_loc = 400
+    y_loc = 250
+    tray_img = make_chip_tray_image(value)
+    table_img.paste(tray_img, (x_loc, y_loc))
+    display_image(canvas, table_img)
+
+def make_chip_tray_image(value):
+    """Return image of chip tray, based on value."""
+    tray_img = Image.new('RGB', (200, 100), (0, 128, 0))
+    ImageDraw.Draw(tray_img).rectangle((0,0,199,99), outline=(255,255,255), fill=None)
+    chip_dia = 40
+    margin = 20
+    chip_step = 5
+    font = ImageFont.truetype('arial.ttf', 20)
+    chip_dict = calc_chips(value)
+    for index, denom in enumerate(DENOMINATIONS):
+        for chip_count in range(chip_dict[denom]):
+            x_loc = margin + index * (chip_dia + margin)
+            y_loc = 40 - chip_count * chip_step
+            chip_box = (x_loc, y_loc, x_loc + chip_dia, y_loc + chip_dia)
+            ImageDraw.Draw(tray_img).ellipse(chip_box, fill=COLORS[index], outline=(0,0,0))
+            (text_x, text_y) = ImageDraw.Draw(tray_img).textsize(denom, font=font)
+            text_loc = (x_loc + 20 - text_x // 2, y_loc + 20 - text_y // 2)
+            ImageDraw.Draw(tray_img).text((text_loc), denom, font=font, fill=(0,0,0))
+    return tray_img
+
+
+def get_card_image(card):
+    """Return the image of the face of a card."""
+    deck_im = Image.open('..\images\Deck.png')
+    row = SUITS.index(card.suit)
+    col = RANKS.index(card.rank)
+    box = (col * CARD_WIDTH, row * CARD_HEIGHT,
+           (col + 1) * CARD_WIDTH, (row + 1) * CARD_HEIGHT)
+    card_im = deck_im.crop(box)
+    return card_im
+
+
+def get_card_back():
+    """Return an image of the back of a card."""
+    deck_im = Image.open('..\images\Deck.png')
+    row = 4
+    col = 5
+    box = (col * CARD_WIDTH, row * CARD_HEIGHT,
+           (col + 1) * CARD_WIDTH, (row + 1) * CARD_HEIGHT)
+    card_im = deck_im.crop(box)
+    return card_im
```

Comment:
You could remove a little duplication and make this process more testable.

This is one of those instances where you could abstract out the calculation parts into a `calc_card_crop_window(card_row, card_col)` function, then be able to doctest that, then wrap it in a `get_card_img(card_row, card_col)` function and delegate to that for `get_card_back()` and `get_card_image()`.